manim/cli/render/commands.py

```diff
@@ -136,4 +139,22 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if nov or args["notify_outdated_version"]:
+        manim = requests.get("https://pypi.org/pypi/manim/json")
+        releases = manim.json()["releases"].keys()
+        latest = max(releases)
+
+        if latest != __version__:
+            console.print(
+                "[yellow]WARNING: You are running on an outdated version of manim[/yellow]"
+            )
+            console.print(f"Installed version: [red]v{__version__}[/red]")
+            console.print(f"Latest version: [green]v{latest}[/green]")
+            console.print(
+                "[yellow]To disable this warning, go to config and set [red]notify_outdated_version[/red] to [/yellow]False"
+            )
+            console.print(
+                "https://docs.manim.community/en/latest/tutorials/configuration.html"
+            )
```

Comment:
```suggestion
```

Reply:
use `stable`

Reply:
I still don't think we should link to this website in the trace. Once `manim cfg` is made better, just like pip has the option to disable the warning with:
`pip config set global.disable-pip-version-check true`
it'd be better to just tell them what `manim cfg` command should be run to disable the warning.